distributed/tests/test_worker.py

```diff
@@ -535,7 +533,63 @@ async def test_spill_to_disk(c, s):
     await x
     assert set(w.data.memory) == {x.key, z.key}
     assert set(w.data.disk) == {y.key}
-    await w.close()
+
+
+requires_zict_210 = pytest.mark.skipif(
+    parse_version(zict.__version__) <= parse_version("2.0.0"),
+    reason="requires zict version > 2.0.0",
+)
+
+
+@requires_zict_210
+@gen_cluster(
+    client=True,
+    nthreads=[("", 1)],
+    worker_kwargs={"memory_limit": 1600, "max_spill": 600},
+)
```

Comment:
You'll hit the spill and pause thresholds.
This test needs to be redesigned; see #5788